packages/flutter/lib/src/widgets/gesture_detector.dart

```diff
@@ -162,6 +169,9 @@ class GestureDetector extends StatelessWidget {
     this.onDoubleTap,
     this.onLongPress,
     this.onLongPressUp,
+    this.onLongPressStart,
```

Comment:
uber-nit: should we sort the longPress callbacks by lifecycle (like the others)? onLongPress, Start, MoveUpdate, Up, End?

Reply:
You didn't like this suggestion? :)

Reply:
oops, missed this one